lib/navigation/view.ts

```diff
@@ -70,6 +70,12 @@ interface ViewData {
 	 * haven't customized their sorting column
 	 */
 	defaultSortKey?: string
+
+	/**
+	 * Method called when the view navigation entry is toggled
```

Comment:
So this is called when the navigation is changed to that view?
Or when you expand the child views?